Develop

cnbrown4 opened the pull request:
Get updated review_response_map

efg commented:
Hi Christine,

Can you tell us which project number this is?  586 is not one of this
year's projects.

Thanks,
[Dr.] Ed Gehringer

On Thu, Oct 29, 2015 at 6:45 PM, cnbrown4 <notifications@github.com> wrote:

> Get updated review_response_map
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/expertiza/expertiza/pull/586
> Commit Summary
>
>    - Updated schema.rb file
>    - Initial commit. Updated README file
>    - Simplifying Import method. Still in progress and untested.
>    - Few changes to make import function smaller. Programming grammar
>    nazified it a little.
>    - Minor grammatical edits.
>    - Made most of the necessary changes. Not thoroughly tested. Adding
>    sprinkles of RoR coding conventions.
>    - Made more changes to make it more Rubyist friendly. Seems to be
>    alright.
>    - Added comments, fixed some grammatical errors.
>    - Added ruby convention to some methods
>    - Added ruby convention to some methods
>    - Added ruby convention to some methods
>    - Added a new private method so shorten a final_versions_from_reviewer
>    method
>    - Added a new private method so shorten a final_versions_from_reviewer
>    method
>    - Added a new private method so shorten a final_versions_from_reviewer
>    method
>    - Removed a few extra lines and comments.
>    - Updated gitignore to ignore the gems directory
>    - Made small changes to keep code climate happy.
>    - Removed extra lines in methods.
>    - Small edits for code climate.
>    - Changes for code climate points.
>    - Code climate change for multi line if statement
>    - Code climate change for multi line if statement. Take 2
>    - Code climate change for multi line if statement. Take 3
>    - Added missing tests
>    - Changes to improve code further.
>    - Changes to improve code.
>    - Merge branch 'develop' of https://github.com/adeeshag/expertiza into
>    develop
>    - Corrected a small error in assignment_nil? function.
>    - Updated the README a little bit. Need to add more.
>
> File Changes
>
>    - *M* .gitignore
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-0> (1)
>    - *M* Gemfile
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-1> (2)
>    - *M* Gemfile.lock
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-2> (136)
>    - *M* README.md
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-3> (29)
>    - *M* app/models/assignment.rb
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-4> (3)
>    - *M* app/models/assignment_participant.rb
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-5> (3)
>    - *M* app/models/review_response_map.rb
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-6> (289)
>    - *D* config/database.yml.example
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-7> (30)
>    - *M* db/schema.rb
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-8> (3)
>    - *M* test/unit/review_response_map_test.rb
>    <https://github.com/expertiza/expertiza/pull/586/files#diff-9> (62)
>
> Patch Links:
>
>    - https://github.com/expertiza/expertiza/pull/586.patch
>    - https://github.com/expertiza/expertiza/pull/586.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expertiza/expertiza/pull/586>.
>

cnbrown4 commented:
Dr. Gehringer,

I made a mistake.  I was having an issue with git hub desktop.  Sorry for
the confusion.


Thank you,
Christine Brown

On Thu, Oct 29, 2015 at 7:23 PM, Ed Gehringer <notifications@github.com>
wrote:

> Hi Christine,
>
> Can you tell us which project number this is? 586 is not one of this
> year's projects.
>
> Thanks,
> [Dr.] Ed Gehringer
>
> On Thu, Oct 29, 2015 at 6:45 PM, cnbrown4 <notifications@github.com>
> wrote:
>
> > Get updated review_response_map
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/expertiza/expertiza/pull/586
> > Commit Summary
> >
> > - Updated schema.rb file
> > - Initial commit. Updated README file
> > - Simplifying Import method. Still in progress and untested.
> > - Few changes to make import function smaller. Programming grammar
> > nazified it a little.
> > - Minor grammatical edits.
> > - Made most of the necessary changes. Not thoroughly tested. Adding
> > sprinkles of RoR coding conventions.
> > - Made more changes to make it more Rubyist friendly. Seems to be
> > alright.
> > - Added comments, fixed some grammatical errors.
> > - Added ruby convention to some methods
> > - Added ruby convention to some methods
> > - Added ruby convention to some methods
> > - Added a new private method so shorten a final_versions_from_reviewer
> > method
> > - Added a new private method so shorten a final_versions_from_reviewer
> > method
> > - Added a new private method so shorten a final_versions_from_reviewer
> > method
> > - Removed a few extra lines and comments.
> > - Updated gitignore to ignore the gems directory
> > - Made small changes to keep code climate happy.
> > - Removed extra lines in methods.
> > - Small edits for code climate.
> > - Changes for code climate points.
> > - Code climate change for multi line if statement
> > - Code climate change for multi line if statement. Take 2
> > - Code climate change for multi line if statement. Take 3
> > - Added missing tests
> > - Changes to improve code further.
> > - Changes to improve code.
> > - Merge branch 'develop' of https://github.com/adeeshag/expertiza into
> > develop
> > - Corrected a small error in assignment_nil? function.
> > - Updated the README a little bit. Need to add more.
> >
> > File Changes
> >
> > - *M* .gitignore
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-0> (1)
> > - *M* Gemfile
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-1> (2)
> > - *M* Gemfile.lock
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-2> (136)
> > - *M* README.md
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-3> (29)
> > - *M* app/models/assignment.rb
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-4> (3)
> > - *M* app/models/assignment_participant.rb
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-5> (3)
> > - *M* app/models/review_response_map.rb
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-6> (289)
> > - *D* config/database.yml.example
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-7> (30)
> > - *M* db/schema.rb
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-8> (3)
> > - *M* test/unit/review_response_map_test.rb
> > <https://github.com/expertiza/expertiza/pull/586/files#diff-9> (62)
> >
> > Patch Links:
> >
> > - https://github.com/expertiza/expertiza/pull/586.patch
> > - https://github.com/expertiza/expertiza/pull/586.diff
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/expertiza/expertiza/pull/586>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expertiza/expertiza/pull/586#issuecomment-152355234>.
>

efg commented:
I installed it myself so I could browse the code while on the plane last
Sunday, but I couldn't figure out how to browse code, just display diffs.
 (I was offline at the time.)

Cheers,
[Dr.] Ed Gehringer

On Thu, Oct 29, 2015 at 9:38 PM, cnbrown4 <notifications@github.com> wrote:

> Dr. Gehringer,
>
> I made a mistake. I was having an issue with git hub desktop. Sorry for
> the confusion.
>
>
> Thank you,
> Christine Brown
>
> On Thu, Oct 29, 2015 at 7:23 PM, Ed Gehringer <notifications@github.com>
> wrote:
>
> > Hi Christine,
> >
> > Can you tell us which project number this is? 586 is not one of this
> > year's projects.
> >
> > Thanks,
> > [Dr.] Ed Gehringer
> >
> > On Thu, Oct 29, 2015 at 6:45 PM, cnbrown4 <notifications@github.com>
> > wrote:
> >
> > > Get updated review_response_map
> > > ------------------------------
> > > You can view, comment on, or merge this pull request online at:
> > >
> > > https://github.com/expertiza/expertiza/pull/586
> > > Commit Summary
> > >
> > > - Updated schema.rb file
> > > - Initial commit. Updated README file
> > > - Simplifying Import method. Still in progress and untested.
> > > - Few changes to make import function smaller. Programming grammar
> > > nazified it a little.
> > > - Minor grammatical edits.
> > > - Made most of the necessary changes. Not thoroughly tested. Adding
> > > sprinkles of RoR coding conventions.
> > > - Made more changes to make it more Rubyist friendly. Seems to be
> > > alright.
> > > - Added comments, fixed some grammatical errors.
> > > - Added ruby convention to some methods
> > > - Added ruby convention to some methods
> > > - Added ruby convention to some methods
> > > - Added a new private method so shorten a final_versions_from_reviewer
> > > method
> > > - Added a new private method so shorten a final_versions_from_reviewer
> > > method
> > > - Added a new private method so shorten a final_versions_from_reviewer
> > > method
> > > - Removed a few extra lines and comments.
> > > - Updated gitignore to ignore the gems directory
> > > - Made small changes to keep code climate happy.
> > > - Removed extra lines in methods.
> > > - Small edits for code climate.
> > > - Changes for code climate points.
> > > - Code climate change for multi line if statement
> > > - Code climate change for multi line if statement. Take 2
> > > - Code climate change for multi line if statement. Take 3
> > > - Added missing tests
> > > - Changes to improve code further.
> > > - Changes to improve code.
> > > - Merge branch 'develop' of https://github.com/adeeshag/expertiza into
> > > develop
> > > - Corrected a small error in assignment_nil? function.
> > > - Updated the README a little bit. Need to add more.
> > >
> > > File Changes
> > >
> > > - *M* .gitignore
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-0> (1)
> > > - *M* Gemfile
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-1> (2)
> > > - *M* Gemfile.lock
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-2> (136)
> > > - *M* README.md
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-3> (29)
> > > - *M* app/models/assignment.rb
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-4> (3)
> > > - *M* app/models/assignment_participant.rb
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-5> (3)
> > > - *M* app/models/review_response_map.rb
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-6> (289)
> > > - *D* config/database.yml.example
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-7> (30)
> > > - *M* db/schema.rb
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-8> (3)
> > > - *M* test/unit/review_response_map_test.rb
> > > <https://github.com/expertiza/expertiza/pull/586/files#diff-9> (62)
> > >
> > > Patch Links:
> > >
> > > - https://github.com/expertiza/expertiza/pull/586.patch
> > > - https://github.com/expertiza/expertiza/pull/586.diff
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/expertiza/expertiza/pull/586>.
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/expertiza/expertiza/pull/586#issuecomment-152355234
> >.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expertiza/expertiza/pull/586#issuecomment-152377122>.
>
